### ISSUE TITLE
Add an example stack file, for documentation

### ISSUE THREAD

**Issue by tomwhoiscontrary** [tim-group/stackbuilder#1]
When trying to extol the virtues of Stackbuilder to people who have not experienced its majesty, it would be really useful to have a stack definition to whip out and demonstrate. It is otherwise very hard to explain what Stackbuilder does, and how easy it makes it to do.

The file should ideally demonstrate:
- a simple single-instance server
- a simple virtual service
- a proxy, NAT, whatever
- have you done something with databases now?
- app-to-app dependencies (this is the coolest bit)
- instantiating stacks in multiple environments (essential to demonstrate the app-to-app bit anyway)
- whatever else is cool

Basically if you could just slightly chop down and and anonymise the internal refstack.rb, that would be great.

For bonus points, attach some app-specific Puppet code that gets hooked in using the mechanism i added. Provided that hasn't already been removed.

**Comment by davidmarkellis** [comment]
I'll try and do that for you, will have time over the weekend.

On Thursday, 14 August 2014, 18:34, Tom Anderson notifications@github.com wrote:

When trying to extol the virtues of Stackbuilder to people who have not experienced its majesty, it would be really useful to have a stack definition to whip out and demonstrate. It is otherwise very hard to explain what Stackbuilder does, and how easy it makes it to do.
The file should ideally demonstrate:
    \* a simple single-instance server
    \* a simple virtual service
    \* a proxy, NAT, whatever
    \* have you done something with databases now?
    \* app-to-app dependencies (this is the coolest bit)
    \* instantiating stacks in multiple environments (essential to demonstrate the app-to-app bit anyway)
    \* whatever else is cool
Basically if you could just slightly chop down and and anonymise the internal refstack.rb, that would be great.
For bonus points, attach some app-specific Puppet code that gets hooked in using the mechanism i added. Provided that hasn't already been removed.

**Comment by rjpearce** [comment]
We have made progress on this but there is much much more for us to do:
Please see: https://github.com/tim-group/stackbuilder-config

Let me know what you think so far.
